arthur-c.rasi

```diff
@@ -63,7 +63,7 @@ configuration {
     highlight:     underline bold #ffffff;
 
     transparent: rgba(0,0,0,0);
-    font: "Ubuntu Mono 16";
```

Comment:
This change seems unrelated.

Reply:
Actually the new layout made some symbols invisible on the right for me, so I decreased the font size a bit to make them fit.

Reply:
Gotchya, understood.